src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -3,5 +3,8 @@
     <type fullname="System.Globalization.GlobalizationMode">
       <method signature="System.Boolean get_Invariant()" body="stub" value="true" feature="System.Globalization.Invariant" featurevalue="true" />
     </type>
+    <type fullname="System.LocalAppContextSwitches">
+      <method signature="System.Boolean get_EnableUnsafeUTF7Encoding()" body="stub" value="true" feature="System.Text.Encoding.EnableUnsafeUTF7Encoding" featurevalue="true" />
```

Comment:
This looks reversed to me. I believe that this would allow UTF7Encoding to be unconditionally enabled; but not trimmed.

Reply:
I think you're right. Even though the default value is _false_, we still want trimming to take place if somebody has set _false_.